### ISSUE TITLE
is school holidays failed

### ISSUE THREAD

**Issue by malys** [tataille/node-red-french-holidays#12]
hi, 
@tataillle thanks you for module.
We are living in Toulouse and currently it's winter holidays but the nodered module returns false.
I think it's a problem of year like previous issue but I don't know this API.
could you check if the last patch has introduced any regression?
thanks

**Comment by tataille** [comment]
Hello
What version of the module?

**Comment by malys** [comment]
1.1.8

**Comment by tataille** [comment]
I will have a look in the next days. Could you check that the version
deployed in nodered is the 1.1.8 (restart)

**Comment by malys** [comment]
I have checked package.json and it's 1.1.8. Moreover ,I have no doubt because at home nodered is shutted down every night and restarted on the morning and I have updated this module in January. is there a way to quickly see URL request?

**Comment by tataille** [comment]
In the logs. I will check your Zone on next week-end and come back to you

**Comment by malys** [comment]
perfect, thanks

**Comment by malys** [comment]
https://data.education.gouv.fr/api/records/1.0/search/?dataset=fr-en-calendrier-scolaire&q=&rows=100&facet=description&facet=start_date&facet=end_date&facet=zones&facet=annee_scolaire&refine.start_date=2023&refine.end_date=2024&refine.location=Toulouse 

```json
{
  "nhits": 1,
  "parameters": {
    "dataset": "fr-en-calendrier-scolaire",
    "rows": 100,
    "start": 0,
    "facet": [
      "description",
      "start_date",
      "end_date",
      "zones",
      "annee_scolaire"
    ],
    "refine": {
      "location": "Toulouse"
    },
    "format": "json",
    "timezone": "UTC"
  },
  "records": [
    {
      "datasetid": "fr-en-calendrier-scolaire",
      "recordid": "e38d78c67194782e90003e72e3d6175823a88092",
      "fields": {
        "end_date": "2024-01-07T23:00:00+00:00",
        "start_date": "2023-12-22T23:00:00+00:00",
        "location": "Toulouse",
        "annee_scolaire": "2023-2024",
        "description": "Vacances de Noël",
        "zones": "Zone C",
        "population": "-"
      },
      "record_timestamp": "2023-10-10T14:00:14.676Z"
    }
  ],
  "facet_groups": [
    {
      "name": "description",
      "facets": [
        {
          "name": "Vacances de Noël",
          "count": 1,
          "state": "displayed",
          "path": "Vacances de Noël"
        }
      ]
    },
    {
      "name": "start_date",
      "facets": [
        {
          "name": "2023",
          "count": 1,
          "state": "refined",
          "path": "2023",
          "facets": [
            {
              "name": "12",
              "count": 1,
              "state": "displayed",
              "path": "2023/12"
            }
          ]
        }
      ]
    },
    {
      "name": "end_date",
      "facets": [
        {
          "name": "2024",
          "count": 1,
          "state": "refined",
          "path": "2024",
          "facets": [
            {
              "name": "01",
              "count": 1,
              "state": "displayed",
              "path": "2024/01"
            }
          ]
        }
      ]
    },
    {
      "name": "zones",
      "facets": [
        {
          "name": "Zone C",
          "count": 1,
          "state": "displayed",
          "path": "Zone C"
        }
      ]
    },
    {
      "name": "annee_scolaire",
      "facets": [
        {
          "name": "2023-2024",
          "count": 1,
          "state": "displayed",
          "path": "2023-2024"
        }
      ]
    },
    {
      "name": "location",
      "facets": [
        {
          "name": "Toulouse",
          "count": 1,
          "state": "refined",
          "path": "Toulouse"
        }
      ]
    }
  ]
}
```

https://data.education.gouv.fr/api/records/1.0/search/?dataset=fr-en-calendrier-scolaire&q=&rows=100&facet=description&facet=start_date&facet=end_date&facet=location&facet=zones&refine.location=Toulouse&refine.annee_scolaire=2023-2024

```json
{
  "nhits": 7,
  "parameters": {
    "dataset": "fr-en-calendrier-scolaire",
    "rows": 100,
    "start": 0,
    "facet": [
      "description",
      "start_date",
      "end_date",
      "location",
      "zones"
    ],
    "refine": {
      "annee_scolaire": "2023-2024"
    },
    "format": "json",
    "timezone": "UTC"
  },
  "records": [
    {
      "datasetid": "fr-en-calendrier-scolaire",
      "recordid": "8ac834925f4e222aa556abdd3c442a8e62bf2a20",
      "fields": {
        "end_date": "2024-09-01T22:00:00+00:00",
        "start_date": "2024-07-05T22:00:00+00:00",
        "location": "Toulouse",
        "annee_scolaire": "2023-2024",
        "description": "Vacances d'Été",
        "zones": "Zone C",
        "population": "Élèves"
      },
      "record_timestamp": "2023-10-10T14:00:14.676Z"
    },
    {
      "datasetid": "fr-en-calendrier-scolaire",
      "recordid": "9dac8a036236e5f5d5a9121b32c1e620fad18a3a",
      "fields": {
        "end_date": "2024-08-29T22:00:00+00:00",
        "start_date": "2024-07-05T22:00:00+00:00",
        "location": "Toulouse",
        "annee_scolaire": "2023-2024",
        "description": "Vacances d'Été",
        "zones": "Zone C",
        "population": "Enseignants"
      },
      "record_timestamp": "2023-10-10T14:00:14.676Z"
    },
    {
      "datasetid": "fr-en-calendrier-scolaire",
      "recordid": "d7b52d527a78946cc59921df59d647155e099729",
      "fields": {
        "end_date": "2024-02-25T23:00:00+00:00",
        "start_date": "2024-02-09T23:00:00+00:00",
        "location": "Toulouse",
        "annee_scolaire": "2023-2024",
        "description": "Vacances d'Hiver",
        "zones": "Zone C",
        "population": "-"
      },
      "record_timestamp": "2023-10-10T14:00:14.676Z"
    },
    {
      "datasetid": "fr-en-calendrier-scolaire",
      "recordid": "38bd569f0f2dcbf79c188284b10cdcb9ce2f9e3a",
      "fields": {
        "end_date": "2024-04-21T22:00:00+00:00",
        "start_date": "2024-04-05T22:00:00+00:00",
        "location": "Toulouse",
        "annee_scolaire": "2023-2024",
        "description": "Vacances de Printemps",
        "zones": "Zone C",
        "population": "-"
      },
      "record_timestamp": "2023-10-10T14:00:14.676Z"
    },
    {
      "datasetid": "fr-en-calendrier-scolaire",
      "recordid": "07ac3788161bf3bdc5cc7ad861dc4e51e2a69cf1",
      "fields": {
        "end_date": "2023-11-05T23:00:00+00:00",
        "start_date": "2023-10-20T22:00:00+00:00",
        "location": "Toulouse",
        "annee_scolaire": "2023-2024",
        "description": "Vacances de la Toussaint",
        "zones": "Zone C",
        "population": "-"
      },
      "record_timestamp": "2023-10-10T14:00:14.676Z"
    },
    {
      "datasetid": "fr-en-calendrier-scolaire",
      "recordid": "e38d78c67194782e90003e72e3d6175823a88092",
      "fields": {
        "end_date": "2024-01-07T23:00:00+00:00",
        "start_date": "2023-12-22T23:00:00+00:00",
        "location": "Toulouse",
        "annee_scolaire": "2023-2024",
        "description": "Vacances de Noël",
        "zones": "Zone C",
        "population": "-"
      },
      "record_timestamp": "2023-10-10T14:00:14.676Z"
    },
    {
      "datasetid": "fr-en-calendrier-scolaire",
      "recordid": "e7cf20d4285a6ef7933021eb547dac455d7e0594",
      "fields": {
        "end_date": "2024-05-10T22:00:00+00:00",
        "start_date": "2024-05-09T22:00:00+00:00",
        "location": "Toulouse",
        "annee_scolaire": "2023-2024",
        "description": "Pont de l'Ascension",
        "zones": "Zone C",
        "population": "-"
      },
      "record_timestamp": "2023-10-10T14:00:14.676Z"
    }
  ],
  "facet_groups": [
    {
      "name": "description",
      "facets": [
        {
          "name": "Vacances d'Été",
          "count": 2,
          "state": "displayed",
          "path": "Vacances d'Été"
        },
        {
          "name": "Pont de l'Ascension",
          "count": 1,
          "state": "displayed",
          "path": "Pont de l'Ascension"
        },
        {
          "name": "Vacances d'Hiver",
          "count": 1,
          "state": "displayed",
          "path": "Vacances d'Hiver"
        },
        {
          "name": "Vacances de Noël",
          "count": 1,
          "state": "displayed",
          "path": "Vacances de Noël"
        },
        {
          "name": "Vacances de Printemps",
          "count": 1,
          "state": "displayed",
          "path": "Vacances de Printemps"
        },
        {
          "name": "Vacances de la Toussaint",
          "count": 1,
          "state": "displayed",
          "path": "Vacances de la Toussaint"
        }
      ]
    },
    {
      "name": "start_date",
      "facets": [
        {
          "name": "2023",
          "count": 2,
          "state": "displayed",
          "path": "2023"
        },
        {
          "name": "2024",
          "count": 5,
          "state": "displayed",
          "path": "2024"
        }
      ]
    },
    {
      "name": "end_date",
      "facets": [
        {
          "name": "2023",
          "count": 1,
          "state": "displayed",
          "path": "2023"
        },
        {
          "name": "2024",
          "count": 6,
          "state": "displayed",
          "path": "2024"
        }
      ]
    },
    {
      "name": "location",
      "facets": [
        {
          "name": "Toulouse",
          "count": 7,
          "state": "refined",
          "path": "Toulouse"
        }
      ]
    },
    {
      "name": "zones",
      "facets": [
        {
          "name": "Zone C",
          "count": 7,
          "state": "displayed",
          "path": "Zone C"
        }
      ]
    },
    {
      "name": "annee_scolaire",
      "facets": [
        {
          "name": "2023-2024",
          "count": 7,
          "state": "refined",
          "path": "2023-2024"
        }
      ]
    }
  ]
}
```

**Comment by tataille** [comment]
Found issue with starting year

**Comment by tataille** [comment]
Fixed in versipn 1.1.9

**Comment by malys** [comment]
thanks :)
